packages/cdktf-cli/package.json

```diff
@@ -27,53 +28,8 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
-    "@apollo/client": "^3.3.21",
-    "@cdktf/hcl2cdk": "0.0.0",
     "@cdktf/hcl2json": "0.0.0",
-    "@graphql-tools/graphql-file-loader": "^6.2.7",
-    "@graphql-tools/load": "^6.2.8",
-    "@skorfmann/ink-confirm-input": "^3.0.0",
-    "@skorfmann/terraform-cloud": "^1.10.1",
-    "@types/node": "^14.0.26",
-    "apollo-server-core": "^3.0.2",
-    "apollo-server-express": "^3.0.2",
-    "archiver": "^5.1.0",
-    "cdktf": "0.0.0",
-    "chalk": "^4.1.0",
-    "chokidar": "^3.5.2",
-    "codemaker": "^0.22.0",
-    "constructs": "^3.3.75",
-    "cross-fetch": "^3.1.4",
-    "date-fns": "^2.22.1",
-    "detect-port": "^1.3.0",
-    "execa": "^5.1.1",
-    "express": "^4.17.1",
-    "extract-zip": "^2.0.1",
-    "follow-redirects": "^1.13.3",
-    "fs-extra": "^8.1.0",
-    "graphql": "^15.5.1",
-    "graphql-subscriptions": "^1.2.1",
-    "indent-string": "^4.0.0",
-    "ink": "^3.0.8",
-    "ink-spinner": "^4.0.1",
-    "inquirer": "^8.1.2",
-    "ink-use-stdout-dimensions": "^1.0.5",
-    "jsii-srcmak": "^0.1.272",
-    "lodash.isequal": "^4.5.0",
-    "log4js": "^6.3.0",
-    "open": "^7.0.4",
-    "parse-gitignore": "^1.0.1",
-    "react": "<17.0.0",
-    "semver": "^7.3.2",
-    "sscaff": "^1.2.0",
-    "stream-buffers": "^3.0.2",
-    "strip-ansi": "^6.0.0",
-    "subscriptions-transport-ws": "^0.9.19",
-    "utility-types": "^3.10.0",
-    "uuid": "^8.3.0",
-    "ws": "^7.5.3",
-    "yargs": "^15.1.0",
-    "zod": "^1.11.7"
+    "cdktf": "0.0.0"
```

Comment:
We should check if this dependency is really necessary

Reply:
As long as we have a global installation of `cdktf-cli` (e.g. via homebrew), it's a bit weird to have this dependency. This could lead to weird behaviour.